site/docs/pages/wallet/wallet-dropdown-link.mdx

```diff
@@ -0,0 +1,90 @@
+import {
+  Address,
+  Avatar,
+  EthBalance,
+  Identity,
+  Name,
+} from '@coinbase/onchainkit/identity';
+import { color } from '@coinbase/onchainkit/theme';
+import {
+  ConnectWallet,
+  Wallet,
+  WalletDropdown,
+  WalletDropdownBaseName,
+  WalletDropdownDisconnect,
+  WalletDropdownLink,
+} from '@coinbase/onchainkit/wallet';
+import WalletComponents from '../../components/WalletComponents';
+
+# `<WalletDropdownLink />`
+
+The `WalletDropdownLink` creates a custom link within the dropdown. Use the icon prop to add an icon, and href to specify the destination.
```

Comment:
For every component is important to teach each props, how to use it.

So in this case we should teach how to use:
- children
-  className
- href
- icon
- rel
- target

I leave it to you decide how best mix and match the demo.